docs/admin/reference/ssh_access.rst

```diff
@@ -0,0 +1,185 @@
+Server SSH Access
+=================
+
+.. _server SSH access:
+
+Server SSH Access
+------------------
```

Comment:
To avoid a duplicate subtitle and make the section more introductory, I would change this subtitle to "Logging in via SSH" or similar, and add a brief reminder that they can log in via the commands `ssh app` or `ssh mon` from the Admin Workstation (future improvement opportunity: troubleshooting SSH access, but we don't have a guide for that yet).